src/php/NamespaceExtractor/Extractor/NamespaceInformation.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\NamespaceExtractor\Extractor;
+
+class NamespaceInformation
+{
+    private string $file;
+    private string $namespace;
+    /** @var string[] */
+    private array $dependencies;
+
+    public function __construct(string $file, string $namespace, array $dependencies)
```

Comment:
We are setting three different properties, but we are using only one "the dependencies" https://github.com/phel-lang/phel-lang/pull/304/files#diff-ca9649ee607c19610338b1cce1a1a09b9e2fa3545eddf22787c15d8dfe0c3dfbR134
Do we need the other properties? Or are we using them somewhere else?

Reply:
I will use them in the Compile Command. I haven't done enough to publish a draft PR for that. But I thought I put out this PR already so it will not get to big.